tests/env-setup/hubSub.bicep

```diff
@@ -0,0 +1,51 @@
+param nameSuffix string = 'blzv'
+
+param location string = 'uksouth'
+
+param vnetAddressSpace string = '192.168.255.0/24'
+
+param vwanHubAddressSpace string = '192.168.252.0/23'
+
+param tags object = {
+  UsedFor: 'Bicep LZ Vending Automated Testing (PRs)'
+  ResponsibleContactAlias: 'jatracey'
```

Comment:
Should we use a security group instead?

Reply:
This is more just for within the CSU Bicep tenant so more of a this is "who" to go harass or to recall who did it 😆